doc/rtd/topics/tutorials/qemu.rst

```diff
@@ -0,0 +1,140 @@
+.. _tutorial_qemu:
+
+Qemu
+****
+
+In this tutorial, we will create our first cloud-init user data config and
+deploy it into a Qemu_ virtual machine. We'll be using Qemu for this tutorial
+which is a popular emulator for running virtual machines on Linux. Several
+popular virtual machine tools use Qemu, including Libvirt, LXD, and Vagrant.
+
+Install Qemu
+============
+
+.. code-block:: sh
+
+    $ sudo apt install qemu-system-x86
+
+See qemu's `install instructions <https://www.qemu.org/download/#linux>`_.
+
+Download a Cloud Image
+======================
+
+All commands should be executed from a single directory; some
+commands create files that are used by later commands. One might want to
+use a temporary directory for easy clean up afterwards.
+
+.. code-block:: sh
+
+    $ wget https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img
+
+Cloud images have cloud-init pre-installed.
+
+
+Define our user data
+====================
+
+Create the following file ``user-data``.
```

Comment:
```suggestion
Create a file called ``user-data`` in your temporary directory and copy/paste the following code into it.
```

Reply:
I've made this confusing, sorry.

The text here describes what the code does, not what the reader is supposed to do. If one creates a file named `user-data` and then copies in the following contents, the tutorial will not work.

The tutorial is written such that all the user should have to do is copy, paste, and execute each code block.

I'll add some clarifying instructions at the top to make this clear.

Reply:
This is probably how I managed to break the tutorial, to be fair...lol